torch/testing/_internal/common_quantization.py

```diff
@@ -450,26 +488,22 @@ def printGraphModule(self, graph_module, print_str=True):
             print(str_to_print)
         return str_to_print
 
-    def checkGraphModeFxOp(self, model, inputs, check_spec, quant_type=QuantType.STATIC, debug=False):
+    def checkGraphModeFxOp(self, model, inputs, quant_type,
+                           expected_node=None,
+                           expected_node_occurrence=None,
+                           expected_node_list=None,
+                           debug=False):
         """ Quantizes model with graph mode quantization on fx and check if the
         quantized model contains the quantized_node
 
         Args:
```

Comment:
optional: since the argument types are non-trivial, would be great to include a short code sample (so code reader can copy-paste from here instead of looking at callsites, reading NodeSpec docs, etc.

Reply:
Sure